### ISSUE TITLE
Bun Add not working with any library

### ISSUE THREAD

**Issue by AlexMercedCoder** [oven-sh/bun#469]
On Pop OS 22.04
Node and Deno work fine

First I tried to install express with `bun add` and it just froze, then tried a bunch of different libraries.

I then upgraded to 0.1.2 (was on 0.1.1) but the issue continued and tried to run add with the `--verbose` flag

I got no real output cause it just froze...

```
😡  bun add react --verbose
bun add v0.1.2
Enqueue package manifest for download: react
  🔍 Resolving [1/1] ^C
```

I waiting a while before killing it and I could here my computer become quite audible like it was doing something. Is there something I haven't tried yet?

**Comment by moyzlevi** [comment]
I guess it should be bun install
![image](https://user-images.githubusercontent.com/14576021/178090064-dcd4582b-5b0c-4002-9918-be3ee4b208c0.png)

Docs: https://github.com/Jarred-Sumner/bun#bun-install

**Comment by hyuzea** [comment]
facing this but not all 
OS: Ubuntu 20.04
 
![image](https://user-images.githubusercontent.com/3823496/178098913-c21b9ce7-2f27-4eb2-940d-af13a61b2268.png)

**Comment by Jarred-Sumner** [comment]
Should be fixed as of Bun v0.1.12. Please comment if it isn't.
